Sources/Mini/Dispatcher.swift

```diff
@@ -133,15 +127,29 @@ public final class Dispatcher {
             if let action = object as? T {
                 completion(action)
             } else {
-                fatalError("Casting to \(tag) failed")
+                print("MiniError: Casting to \(tag) failed")
```

Comment:
This must crash because data revving has been broken, if this happens, something about threading is wrong.

Reply:
Ok, replaced. I like to control errors rather than cause crashes, but it doesn't seem to be a frequent error case.